### PR TITLE
Update 14.biological-interpretation.md

### PR DESCRIPTION
**Pull request by ger225**
Hi, I am working on this part. I have uploaded a description of network analysis, and I will continue working on network enrichment and the references. Germán Rosano

Please provide a description of your changes:

Please check the following about your suggested changed:
- [x] The new text has one sentence per line
- [x] preprints that are cited are listed as an issue for discussion and peer review
network3

### CODE REVIEW

**Comment by jessegmeyerlab** [comment]
Thanks @ger225 ! This looks like a great start. 

There are three main changes we need here:
1. add citations. The easiest way is to do this in pubmed IDs with this format: [@PMID:25479658], where the number is the pubmed id of the paper you wish to cite. You can also do this with @DOI: if its easier for you to find the DOI of the paper you want to cite
2. Each sentence should be its own line. The reason for this is that it makes it easy to see which sentences change in edits because github tracks line changes. Right now if someone wants to change a sentence it would appear the whole paragraph changes
3. It looks like you want to add a figure you made? Please check out the format for adding figures in another section that has a figure. Basically you need to upload the figure to the folder /content/images/ and then you add a line in your section referring to that figure. For example you can look at the raw text in this section where the figure is added: https://github.com/jessegmeyerlab/proteomics-tutorial/blob/main/content/05.proteolysis.md

**Comment by ger225** [comment]
Thanks for the pointers Dr. Meyer. I will start working on them right away. Best,

**Comment by jessegmeyerlab** [comment]
Looks like you are making more progress this is awesome thank you. Please ping me when you are done and want me to do a review

**Comment by ger225** [comment]
As I wrote in another email, the section is up for review.

Best wishes,

--
Dr. Germán L. Rosano

Instituto de Biología Molecular y Celular de Rosario (IBR - CONICET),
Esmeralda y Ocampo,  (2000) Rosario, Argentina.
Tel: (0341) 4237070 Int.: 665

File: content/14.biological-interpretation.md [hunk 1]
```diff
@@ -1,8 +1,132 @@
-## Biological Interpretation {.page_break_before}
+## Biological Interpretation
 
+As described in previous sections, quantitative proteomics experiments reveal protein abundance changes. 
+After statistical data analysis, the typical output is a list of differentially expressed proteins that can contain hundreds of hits. 
+The next step is how to interpret the results in biological terms. 
+Often, it is tempting to focus on proteins with the most extreme fold changes. 
+In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior. 
+Alas, this assumption is not always valid. 
+Another approach is to start googling for every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
Another approach is to start manually lookup every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways. 
```

File: content/14.biological-interpretation.md [hunk 2]
```diff
@@ -1,8 +1,132 @@
...
+## Biological Interpretation
 
+As described in previous sections, quantitative proteomics experiments reveal protein abundance changes. 
+After statistical data analysis, the typical output is a list of differentially expressed proteins that can contain hundreds of hits. 
+The next step is how to interpret the results in biological terms. 
+Often, it is tempting to focus on proteins with the most extreme fold changes. 
+In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior. 
+Alas, this assumption is not always valid. 
+Another approach is to start googling for every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways. 
+This strategy is time-consuming and cumbersome, and it is almost impossible to capture all reported interactions from the wealth of the literature.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md:10]:
```suggestion
This strategy is time-consuming and cumbersome, and it is almost impossible to capture all reported interactions from the wealth of the literature. 
Therefore, automated computational methods to help with biological interpretation are needed. 
In this section, we overview some of these methods. 
```

File: content/14.biological-interpretation.md [hunk 3]
```diff
@@ -1,8 +1,132 @@
...
 
-1. term enrichment analysis (KEGG, GO)
-2. network analysis methods
-3. structure analysis
-4. isoform analysis
-5. follow-up experiments
+### Constructing a protein network
+
+A network is a representation of the relations between objects. 
+Nodes are the entities of the network (users of a social platform, train stations, proteins), while edges are the connections between them (friendship, routes, and protein interactions, respectively).
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
Nodes are the entities of the network (e.g., users of a social platform, train stations, proteins), while edges are the connections between them (e.g., friendship, routes, and protein interactions, respectively). 
```

File: content/14.biological-interpretation.md [hunk 4]
```diff
@@ -1,8 +1,132 @@
...
+In the case of protein-protein interactions, the evidence for the functional associations between proteins can be obtained experimentally. 
+For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions [@PMID:34046695]. 
+The data is presented in a table with nodes and edges (e.g., "protein A interacts with protein B") from which the network can be constructed. 
+A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions [@PMID:34761267]. 
+Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction [@PMID:21572181; @PMID:33237311]. 
+Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases [@PMID:33237311]. 
+It also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
STRING also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 
```

File: content/14.biological-interpretation.md [hunk 5]
```diff
@@ -1,8 +1,132 @@
...
+A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions [@PMID:34761267]. 
+Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction [@PMID:21572181; @PMID:33237311]. 
+Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases [@PMID:33237311]. 
+It also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 
+STRING will retrieve the network and show the evidence supporting each interaction. 
+Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networks. 
+If edges have directions (e.g., A influences B and not vice versa), then the network is directed. Signaling pathways are examples of directed graphs.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
If edges have directions (e.g., A influences B and not vice versa), then the network is directed. 
Signaling pathways are examples of directed graphs. 
```

File: content/14.biological-interpretation.md [hunk 6]
```diff
@@ -1,8 +1,132 @@
...
+
+#### Degree centrality
+  
+The degree of a node measures the number of edges incident to that node. 
+Nodes with a high degree interact with many other nodes, called first neighbors. 
+In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
+Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
+Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network [@PMID:16751849]. 
+
+#### Closeness centrality
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md:44]: can you add a citation to the text below about closeness centrality where people can look for more information?

File: content/14.biological-interpretation.md [hunk 7]
```diff
@@ -1,8 +1,132 @@
...
+Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task.
+
+#### Betweenness centrality
+
+This centrality index is related to the amount of shortest paths transversing a node. 
+Nodes with a high betweenness centrality usually bridge different parts of the network and strongly influence the flow of information, as they lie in communication paths. 
+These connector hubs (or bottlenecks) are also interesting for follow–up experiments because their removal can disconnect different regions of the network [@PMID:17447836].
+
+Centrality measurements add new layers of information and allow for ranking differentially expressed proteins apart from their fold-change in abundance. 
+Figure X depicts a simple network consisting of proteins A to L, with A having the highest fold change (10) and L the lowest (2).
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]: This figure is awesome thank you

File: content/14.biological-interpretation.md [hunk 8]
```diff
@@ -1,8 +1,132 @@
...
+Networks can rapidly become a hairball unamenable to interpretation. 
+Software platforms like Cytoscape can be used to visualize networks orderly by applying layout algorithms and format styles [@PMID:14597658]. 
+Since many of these platforms are open source, community-designed plugins enhance their capabilities. 
+In Cytoscape, the stringApp adds a search bar to query the STRING database with accession numbers or protein names [@PMID:36512705]. 
+The network is directly retrieved into Cytoscape, where its built-in network analyzer can be used to calculate centralities. 
+Moreover, user-defined information, like fold-change values, can be integrated and mapped into the network.
+
+### Functional enrichment analysis
+
+Gene and gene products have shared attributes regarding their cellular localization, molecular function, or biological process in which they take part, among others.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
Genes and gene products have shared attributes regarding their cellular localization, molecular function, or biological process in which they take part.
```

File: content/14.biological-interpretation.md [hunk 9]
```diff
@@ -1,8 +1,132 @@
...
+For example, citrate synthase and malate dehydrogenase are _mitochondrial_ enzymes that show _RNA binding_ activity and participate in the _Krebs cycle_. 
+The terms in italics represent a fraction of the body of knowledge about these proteins. 
+Similar information can be compiled and assigned to every gene or gene product, forming its gene ontology (GO): a set of concepts that describe the entity. 
+Functional enrichment analysis aims to reveal if genes or gene products grouped under a GO term (a.k.a., a "gene set") are over- or under-represented in a more extensive set.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
Functional enrichment analysis assesses whether genes or gene products grouped under a GO term (a.k.a., a "gene set") are overrepresented in the set of proteins of interest compared to their normal frequency of occurrence in some background dataset. 
This is commonly achieved using Fisher's exact test [@DOI:10.2307/2340521].
```

File: content/14.biological-interpretation.md [hunk 10]
```diff
@@ -1,8 +1,132 @@
...
+The terms in italics represent a fraction of the body of knowledge about these proteins. 
+Similar information can be compiled and assigned to every gene or gene product, forming its gene ontology (GO): a set of concepts that describe the entity. 
+Functional enrichment analysis aims to reveal if genes or gene products grouped under a GO term (a.k.a., a "gene set") are over- or under-represented in a more extensive set. 
+For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels [@PMID:33060801]. 
+Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle. 
+GO terms [@PMID:10802651] or descriptors from other annotation libraries (like KEGG [@PMID:10592173] or REACTOME [@PMID:34788843]) can be retrieved from STRING when constructing a network or from other freely available compendiums. 
+Then, it is a matter of conducting a functional enrichment test to analyze which terms are over- or under-represented.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]: Over-representation analysis is very common but I'm not aware of under-representation analysis, do you have an example of that? If not can we change to just over-representation?

File: content/14.biological-interpretation.md [hunk 11]
```diff
@@ -1,8 +1,132 @@
...
+
+#### Over-representation analysis
+  
+In a proteomics analysis, thousands of proteins are identified and quantified.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
In a modern proteomics analysis, usually thousands of proteins are identified and quantified. 
```

File: content/14.biological-interpretation.md [hunk 12]
```diff
@@ -1,8 +1,132 @@
...
+The terms in italics represent a fraction of the body of knowledge about these proteins. 
+Similar information can be compiled and assigned to every gene or gene product, forming its gene ontology (GO): a set of concepts that describe the entity. 
+Functional enrichment analysis aims to reveal if genes or gene products grouped under a GO term (a.k.a., a "gene set") are over- or under-represented in a more extensive set. 
+For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels [@PMID:33060801]. 
+Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle. 
+GO terms [@PMID:10802651] or descriptors from other annotation libraries (like KEGG [@PMID:10592173] or REACTOME [@PMID:34788843]) can be retrieved from STRING when constructing a network or from other freely available compendiums. 
+Then, it is a matter of conducting a functional enrichment test to analyze which terms are over- or under-represented.
+
+#### Over-representation analysis
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md:103]: this section seems redundant with the previous section - can you combine the two?

File: content/14.biological-interpretation.md [hunk 13]
```diff
@@ -1,8 +1,132 @@
...
+
+	                    
+A Fisher's exact test, or another suitable statistical test, is conducted to ascertain if proteins with altered levels are enriched in members of the TCA cycle (in this case, they are; p < 0.00001). 
+This process is then repeated for every set as desired. 
+Since multiple comparisons are tested, p values must be adjusted by a false discovery rate.
+
+#### Functional class scoring
+  
+The caveat of over-representation methods is that they rely on a list of differentially expressed genes or proteins with altered abundance, selected due to arbitrarily chosen cut-off values. 
+So, a significant hit with a fold-change of 1.99 would not be included in the analysis.
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
For example, if we set a fold change cutoff of 2, a protein with a fold-change of 1.99 would not be included in the analysis. 
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
I think this is almost perfect thank you. Please take a look at my requested changes.

File: content/14.biological-interpretation.md (hunk 6 above)
Comment (ger225) [reply content/14.biological-interpretation.md:44]: Closeness centrality is not as heavily used as betweenness in protein networks. Still, I could add a nice citation showing that essential genes in yeast have a higher closeness than non-essential ones. I also added a link to a great resource at the European Bioinformatics Institute, but I don´t know if citing web pages is appropriate for this manuscript.

File: content/14.biological-interpretation.md (hunk 10 above)
Comment (ger225) [reply content/14.biological-interpretation.md]: Pardon my wording. For under-representation, I meant depletion. For clarity's sake, it is best to stick with over-representation for an introductory level.

File: content/14.biological-interpretation.md (hunk 12 above)
Comment (ger225) [reply content/14.biological-interpretation.md:103]: I kept the three sub-sections as this better reflects the three most widely used types of functional enrichment methods (ORA, FCS, and PT). Instead, I tried to reduce redundancy by rewriting part of the introductory paragraph.

File: content/14.biological-interpretation.md [hunk 14]
```diff
@@ -1,8 +1,139 @@
-## Biological Interpretation {.page_break_before}
+## Biological Interpretation
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
## Biological Interpretation  {.page_break_before}
```

File: content/14.biological-interpretation.md [hunk 15]
```diff
@@ -1,8 +1,139 @@
...
+In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
+Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
+Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network [@PMID:16751849]. 
+
+#### Closeness centrality
+
+The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps. 
+Closeness centrality is the inverse of the average length of a node's shortest paths to all other nodes in the network. 
+Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task [@PMID:15616139; @https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/].
```
Comment (jessegmeyerlab) [inline content/14.biological-interpretation.md]:
```suggestion
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task [@PMID:15616139; @URL:https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/].
```

**Comment by jessegmeyerlab** [comment]
OK I think its merged!!! Thanks German @ger225 - please make a new pull request adding your name and details to the metadata.yml so we can be sure you are in the author list

**Comment by ger225** [comment]
Awesome, thanks!

--
Dr. Germán L. Rosano

Instituto de Biología Molecular y Celular de Rosario (IBR - CONICET),
Esmeralda y Ocampo,  (2000) Rosario, Argentina.
Tel: (0341) 4237070 Int.: 665


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Remitente
notificado con
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
08/01/23,
09:21:49

**Pull request merged**